README.md

```diff
@@ -3,13 +3,104 @@
 A set of Babel plugins that enable injecting different polyfills with different strategies in your compiled code.
 Additionally, this reporitory contains a package that helps creating providers for any other polyfill.
 
-> ⚠️ These packages are highly experimental and not published yet. You can use GitHub's "Watch for Releases only" feature to be notified when the first version is published.
+> ⚠️ These packages are **highly experimental** and they have not been tested in production applications yet. Also, we are still working ond wiring some missing polyfills.
 
 > ℹ️ This repository implements what was initially proposed at [babel/babel#10008](https://github.com/babel/babel/issues/10008).
 
 > 💡 If you are looking for some quick setup examples, or just want to see how to migrade your config, please check [`docs/migration.md`](https://github.com/babel/babel-polyfills/blob/master/docs/migration.md).
 
-## How does it work?
+## Usage
+
+The main Babel packages only transform JavaScript _syntax_: you also need to load a polyfill, to make native _functions_ (`Array.prototype.flat`) or _built-in objects_ (`Promise`) work in older browsers.
+
+The easiest way to do so is to directly load the polyfill using a `<script ...>` tag:
+```html
+<script src="https://unpkg.com/core-js-bundle@3.6.5/minified.js"></script>
+```
+
+However, this simple approach can potentially include a lot of unnecessary code. The Babel plugins implemented in this repository automatically inject the polyfills in your code, while trying to only load what is really needed. It does this based on your compilation targets and on what you are using in your code.
```

Comment:
Maybe everything before `The Babel plugins implemented in this repository automatically ` could go in a `## Motivation/Reasoning/Why Section`?

Reply:
We have that section at the end, but I think that a few sentence i this Usage section (since it is what most people read) help giving a bit of context before reading everything without distracting too much with the long explanation.